src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py

```diff
@@ -109,8 +113,11 @@ def on_event_start(
         )
         event_data["attributes"] = {}
         # Parse the payload to extract the parameters
-        if payload is not None:
-            event_data["attributes"].update(payload_to_semantic_attributes(payload))
+        try:
+            if payload is not None:
+                event_data["attributes"].update(payload_to_semantic_attributes(payload))
+        except Exception:
```

Comment:
What exceptions do we expect? Shouldn't the exception catcher be inside `payload_to_semantic_attributes`, or should this just make use of a narrower `if` statement?
```python
if isinstance(payload, dict):
```

Reply:
Whoops, I made this change while debugging an issue and meant to change it back. Addressed in e1756c1. Good catch.